plugins/BEdita/Core/src/Model/Table/TreesTable.php

```diff
@@ -0,0 +1,112 @@
+<?php
+namespace BEdita\Core\Model\Table;
+
+use Cake\ORM\Query;
+use Cake\ORM\RulesChecker;
+use Cake\ORM\Table;
+use Cake\Validation\Validator;
+
+/**
+ * Trees Model
+ *
+ * @property \Cake\ORM\Association\BelongsTo $Objects
+ * @property \Cake\ORM\Association\BelongsTo $ParentTrees
+ * @property \Cake\ORM\Association\BelongsTo $Objects
+ * @property \Cake\ORM\Association\HasMany $ChildTrees
+ *
+ * @method \BEdita\Core\Model\Entity\Tree get($primaryKey, $options = [])
+ * @method \BEdita\Core\Model\Entity\Tree newEntity($data = null, array $options = [])
+ * @method \BEdita\Core\Model\Entity\Tree[] newEntities(array $data, array $options = [])
+ * @method \BEdita\Core\Model\Entity\Tree|bool save(\Cake\Datasource\EntityInterface $entity, $options = [])
+ * @method \BEdita\Core\Model\Entity\Tree patchEntity(\Cake\Datasource\EntityInterface $entity, array $data, array $options = [])
+ * @method \BEdita\Core\Model\Entity\Tree[] patchEntities($entities, array $data, array $options = [])
+ * @method \BEdita\Core\Model\Entity\Tree findOrCreate($search, callable $callback = null, $options = [])
+ */
+class TreesTable extends Table
+{
+
+    /**
+     * Initialize method
+     *
+     * @param array $config The configuration for the Table.
+     * @return void
+     */
+    public function initialize(array $config)
+    {
+        parent::initialize($config);
+
+        $this->table('trees');
+        $this->displayField('id');
+        $this->primaryKey('id');
+
+        $this->belongsTo('Objects', [
```

Comment:
Same as above. Conflict in `belongsTo` assoications using the same alias `Objects`